pkgs/applications/audio/spotify-cli/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, python3Packages, fetchFromGitHub, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "spotify-cli";
+  version = "0.3.12";
+
+  src = fetchFromGitHub {
+    owner = "ledesmablt";
+    repo = pname;
+    rev = "8e0ad476079a722a39dd2072195c6b4562d18a4d";
+    sha256 = "sha256-KSlv+O1XQPFrRyxx8/+lCzWryp+9aXpB5CvhEM9w1wA=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    click
+    inquirerpy
+    tabulate
+  ];
+
+  preBuild = ''
+    export RELEASE_VERSION=${version}
+  '';
+
+  doCheck = false;
+
+  meta = with lib; {
```

Comment:
You should add a [`mainProgram` attribute](https://github.com/NixOS/nixpkgs/blob/826ad52cfc73623af389932cb84b000f315b1211/doc/stdenv/meta.chapter.md?plain=1#L72), since there is a clear executable that we can expose via `nix run`.